doc/main.html

```diff
@@ -230,50 +290,63 @@ <h4>Conformance</h4>
 
         <p>The <code>id</code> attribute shall be present and equal to <code>sec-conformance</code>.</p>
 
+        <p>If no additional user text is required, the section shall not be present.</p>
+
         <p>The heading will be generated automatically.</p>
       </section>
 
       <section id="sec-norm-refs-section">
-        <h4>Normative references</h4>
+        <h4>Normative references section</h4>
 
-        <p>The <dfn>normative references section</dfn> contains a single unordered list <code>ul</code> where each element
+        <p>The <dfn>Normative references</dfn> contains a single unordered list <code>ul</code> where each element
         <code>li</code> is a normative reference.</p>
 
+        <p>If using normative references, the section shall be present as: </p>
+
 <pre>&lt;section id=&quot;sec-normative-references&quot;&gt;
   &lt;ul&gt;
     &lt;li&gt;
-      &lt;cite id=&quot;bib-ietf-rfc-3986&quot;&gt;IETF RFC 3986&lt;/cite&gt;, Uniform Resource Identifiers (URI): Generic Syntax,
-      T. Berners-Lee, et al., January 2005. &lt;a&gt;http://www.ietf.org/rfc/rfc3986.txt&lt;/a&gt;
+      &lt;cite id=&quot;bib-HTML-5&quot;&gt;HTML Standard&lt;/cite&gt;, Living Standard. 
+        &lt;a&gt;https://html.spec.whatwg.org/multipage/&lt;/a&gt;&lt;/li&gt;
     &lt;/li&gt;
   &lt;/ul&gt;
 &lt;/section&gt;</pre>
 
-        <p>The <code>id</code> attribute shall be present on the <code>section</code> element and equal to
+        <p>The <code>id</code> attribute shall be present and equal to
```

Comment:
Why not specify where is it present?

Reply:
I think that was a bad copy/paste. Thanks.